sender.go

```diff
@@ -804,6 +810,13 @@ func (s *LineSender) Flush(ctx context.Context) error {
 	return nil
 }
 
+func (s *LineSender) discardPendingMsg() {
+	s.buf.Truncate(s.lastMsgPos)
+	s.hasTable = false
```

Comment:
nit: Is it worth refactoring [these 3 lines](https://github.com/questdb/go-questdb-client/blob/2890ab58ca3a7c707aa8fa52525436484300a17d/sender.go#L757-L759) into a function, since they're also used verbatim here in your change?

Reply:
Addressed in 68d8e35c1c2e60397f041193ba1f1d6991e58048